### ISSUE TITLE
Consonants in name generators

### ISSUE THREAD

**Issue by Wedge009** [wesnoth/wesnoth#9607]
Looking at https://github.com/wesnoth/wesnoth/blob/master/data/core/macros/names.cfg from the perspective of a translator, it's not immediately clear to me why 'consonant' is repeatedly mis-spelled with four 'n' characters. Is it just a case of incorrect spelling in the original development (akin to HTTP 'referer') or is there an actual reason for using wrong spelling?

**Comment by soliton-** [comment]
It's there since the name generator was added in f84e8f4. I guess it's a typo. It's just an internal identifier anyway. Only `main` is required.

**Comment by Wedge009** [comment]
Yeah, I noted the merged PR by Dugy. I suppose I just wanted to confirm it wasn't anything more than a spelling mistake, because the entire generator is given for translation. Well, hopefully the translators recognise this. Is it worth adding a translation hint or just fixing the generator expressions?

**Comment by soliton-** [comment]
Fixing the expression probably just means a lot of fuzzy strings for minimal gain. If we'd use pofix this would be a good time...

Adding a translation hint about the spelling mistake does not seem particularly useful. Even if a translation fixes the spelling it doesn't change anything?

**Comment by spixi** [comment]
The translation can use a completely different grammar. You can name your nonterminals like you want.

**Comment by Wedge009** [comment]
I suppose the question is answered, it's a non-user-visible error that may only be a potential issue for translators.
